src/dag-node.js

```diff
@@ -53,7 +53,7 @@ function DAGNode (data, links) {
   this.addRawLink = (link) => {
     encoded = null
     this.links.push(new DAGLink(link.name, link.size, link.hash))
-    stable.inplace(this.links,linkSort)
+    stable.inplace(this.links)
```

Comment:
was flagged in standard for no space so i took it off and yielded that same results without a comparator function. Not sure why so putting it back.